fluentinterface/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: pattern
+title: Fluent Interface
+folder: fluentinterface
+permalink: /patterns/fluentinterface/
+categories: Architectural
+tags: Java
+---
+
+**Intent:** A fluent interface provides an easy-readable, flowing interface, that often mimics a domain specific language. This is especially useful when the users of your API benefit from code, that can nearly be read as human language.
```

Comment:
Of course, if one has a better idea how to express it - english is not my native language ;)

I wanted to express that this kind of interface is most useful for people that really want to have an easy API, because they're not interested further in the topic of your program, but want to utilize it as quickly as possible. Or when the fluent interface design would really be much better then a traditional approach.